source/includes/steps-install-shell-linux-deb.yaml

```
@@ -93,25 +93,23 @@ level: 4
 ref: install
 content: |
 
-  To install the latest stable version of ``mongosh``, issue the
-  following command:
+  ``mongosh`` supports OpenSSL. You can install ``mongosh`` without
+  OpenSSL or you can use your system's OpenSSL installation to enhance
```

Super small grammar nit

```suggestion
  OpenSSL, or you can use your system's OpenSSL installation to enhance
```

---

Done